prisma/schema.prisma

```diff
@@ -25,22 +33,30 @@ model Password {
   hash   String
 }
 
-// Originally thought it was going to be a many-to-many relationship
-// Thought about it again and decided it was one-to-many
-// Andre - looking forward to your feedback!
 model Group {
-  id          String @id @default(uuid())
+  id          String  @id @default(uuid())
   name        String
   description String
   events      Event[]
+  users       User[]
+}
+
+model User_Group {
+  userId  String
+  groupId String
+  role    Role
+
+  @@id([userId, groupId])
```

Comment:
The primary key of the `User_Group` table is defined by the combination of `userId` and `groupId`. This means each `userId`-`groupId` combination has to be unique.